src/app/components/challenge/challengephases/phasecard/phasecard.component.ts

```diff
@@ -81,10 +81,18 @@ export class PhasecardComponent implements OnInit {
     SELF.apiCall = (params) => {
       const FORM_DATA: FormData = new FormData();
       for (const key in params) {
-        if (params[key]) {
-          FORM_DATA.append(key, params[key]);
+        if (params['max_submissions_per_day'] <= params['max_submissions_per_month'] &&
```

Comment:
Can we have separate method for these kind of logics in some services, so that we follow DRY ? Try make method length lesser.